consensus/blake3pow/consensus.go

```diff
@@ -320,7 +320,12 @@ func (blake3pow *Blake3pow) CalcDifficulty(chain consensus.ChainHeaderReader, pa
 
 	// (2 if len(parent_uncles) else 1) - (block_timestamp - parent_timestamp) // duration_limit
 	x.Sub(bigTime, bigParentTime)
-	x.Div(x, params.DurationLimit[nodeCtx])
```

Comment:
I would strongly prefer it if `params.DurationLimit[]` was set dynamically according to the chain config. Perhaps its a parameter in `Blake3pow.Config` which gets set at node startup according to the duration limit given by the chain config?